mempool/src/verify.rs

```diff
@@ -71,6 +73,9 @@ impl Display for VerifyErr {
             VerifyErr::Filtered => {
                 write!(f, "Filtered")
             }
+            VerifyErr::CantSucceed => {
+                write!(f, "Cant succeed")
```

Comment:
Either `Cannot` or `Can't`. I'm personally in favor of `Cannot` and would also change the error to be called @'CannotSucceed`.